pyfstat/make_sfts.py

```diff
@@ -742,6 +742,10 @@ def make_data(self, verbose=False):
         self.run_makefakedata()
 
     def run_makefakedata(self):
+        mfd_command_line = self.build_MFD_command_line()
+        self._execute_MFD(mfd_command_line)
```

Comment:
Actually it seems you ended up only needing to override `build_MFD_command_line` for the new class, so the contents of `_execute_MFD` could just be moved back into this one?

Reply:
I though for a second to leave this as a backdoor to hijack the MFD command line, but that can be done as easily without both methods, so out.